### ISSUE TITLE
[Bug]: Desktop taskbar icon does not flash / blink when receiving messages

### ISSUE THREAD

**Issue by tracewave** [simplex-chat/simplex-chat#3301]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Windows

### OS version

Windows 11

### App version

v5.4.0-beta.2

### Current Behavior

Currently, when I receive new messages in the application, the icon in the Windows taskbar remains static and does not exhibit any visual cues to indicate incoming messages. This makes it difficult to promptly notice and respond to messages, especially when the application is running in the background.

### Expected Behavior

When new messages are received, the application's taskbar icon should start flashing, following the standard Windows convention for indicating new messages or notifications. Additionally, the icon's color should change to signify the presence of unread messages, and this color change should persist until the application window gains focus or the messages are marked as read.

This should happen even if the chat with the new message is opened or not, while the SimpleX window is out of focus.

Image example (from internet): 
![Windows 10 example](https://i.imgur.com/jOmMgcU.png)

### Steps To Reproduce

1. Focus another window.
2. Receive a message with or without the chat opened.
3. The taskbar icon does not flash.

**Comment by ITwrx** [comment]
yeah, I'm using xfce on f39. gajim uses a separate tray/panel icon that changes when you have unread messages while the main app icon doesn't change. I would be happy with either a tray icon that changes like gajim, or if the app window icon can change, but i'm not sure if the latter is possible. I don't remember how the window app icons are implemented and none of them are currently behaving dynamically/changing due to state on my DE, so i'm thinking that may not be supported and tray/panel icon pair may be necessary.

**Comment by deGoyan** [comment]
> I can add to the same suggestion, to change the name of the window to indicate amount of unread messages, like: SimpleX (10)
Like that
![image](https://github.com/user-attachments/assets/34c8d747-9fcd-48e7-81ee-9b354c02017d)
